<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Бюро недвижимости</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="table">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
        <column>
         <property name="text">
          <string>Код</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Тип</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Кол-во комнат</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Площадь</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Город</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Цена</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Фильтры</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Тип объекта</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_type">
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Дом</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Квартира</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Дача</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Максимальная цена</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sb_price">
           <property name="maximum">
            <number>1000000000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="bt_filter">
           <property name="text">
            <string>Применить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bt_reset">
           <property name="text">
            <string>Сбросить</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Действие</string>
    </property>
    <addaction name="ac_add"/>
    <addaction name="ac_del"/>
    <addaction name="separator"/>
    <addaction name="ac_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ac_add">
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="ac_del">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="ac_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
